testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2HttpProxy.java

```diff
@@ -82,9 +82,5 @@ public void extendEnvironment(Environment.Builder builder)
         builder.containerDependsOn(COORDINATOR, hydraClientConfig.getLogicalName());
 
         builder.containerDependsOn(COORDINATOR, PROXY);
-        builder.configureContainer(PROXY, dockerContainer -> {
-            dockerContainer
-                    .withCopyFileToContainer(forHostPath(configDir.getPath("httpd.conf")), "/usr/local/apache2/conf/httpd.conf");
-        });
```

Comment:
This change has to be squahsed to the previous one right ?

Reply:
yes,  done